shopifyHmacExample.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const hmac = require('.');
+const extractSignature = require('./lib/shopifyHmac/extractHmacParameter');
+const constructSignatureString = require('./lib/shopifyHmac/constructHmacParameter');
+
+module.exports = function (fastify, options, next) {
+  fastify.register(hmac, {
+    sharedSecret: 'hush',
+    verificationErrorMessage: 'Shopify HMAC parameter verification failed',
+    extractSignature,
+    constructSignatureString,
+    getAlgorithm: () => 'sha256',
+    getDigest: () => 'hex',
+  });
```

Comment:
I would import these as their actual method names (although see other comment about the method names), and then not use the shorthand. I think it's clearer what is happening for the example:

```suggestion
const extractHmacParameter = require('./lib/shopifyHmac/extractHmacParameter');
const constructHmacParameter = require('./lib/shopifyHmac/constructHmacParameter');

module.exports = function (fastify, options, next) {
  fastify.register(hmac, {
    sharedSecret: 'hush',
    verificationErrorMessage: 'Shopify HMAC parameter verification failed',
    extractSignature: extractHmacParameter,
    constructSignatureString: constructHmacParameter,
    getAlgorithm: () => 'sha256',
    getDigest: () => 'hex',
  });
```